website/docs/r/virtual_machine_scale_set.html.markdown

```diff
@@ -129,10 +129,11 @@ resource "azurerm_virtual_machine_scale_set" "test" {
     primary = true
 
     ip_configuration {
-      name                                   = "TestIPConfiguration"
-      subnet_id                              = "${azurerm_subnet.test.id}"
-      load_balancer_backend_address_pool_ids = ["${azurerm_lb_backend_address_pool.bpepool.id}"]
-      load_balancer_inbound_nat_rules_ids    = ["${element(azurerm_lb_nat_pool.lbnatpool.*.id, count.index)}"]
+      name                                         = "TestIPConfiguration"
+      subnet_id                                    = "${azurerm_subnet.test.id}"
+      application_gateway_backend_address_pool_ids = ["${azurerm_application_gateway.mygateway.backend_address_pool.id}"]
```

Comment:
given this isn't present in the example (and I believe it conflicts with the LB fields?), it might be prudent to comment this out as an example?

Reply:
I see your point. While it's not common, they're actually not mutually exclusive. You could techinically wire your VMSS with both a load balancer (for NAT rules or internal LBs) and an application gateway (for public HTTP traffic).

Reply:
I commented it out anyway.

Reply:
thanks for confirming that - I guess we can leave it commented out for the moment either way :)